tests/federation/test_federation_sender.py

```diff
@@ -343,7 +343,9 @@ def test_send_device_updates(self) -> None:
         self.reactor.advance(1)
 
         # a second call should produce no new device EDUs
-        self.hs.get_federation_sender().send_device_messages("host2")
+        self.get_success(
+            self.hs.get_federation_sender().send_device_messages(["host2"])
+        )
```

Comment:
Ugh, more of these. Maybe we should resurrect https://github.com/matrix-org/synapse/pull/14519 , at least for the tests